orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationChatResponse.java

```diff
@@ -38,4 +42,24 @@ public String getContent() throws OrchestrationClientException {
     }
     return choice.getMessage().getContent();
   }
+
+  /**
+   * Get the token usage.
+   *
+   * @return The token usage.
+   */
+  @Nonnull
+  public TokenUsage getTokenUsage() {
+    return ((LLMModuleResultSynchronous) originalResponse.getOrchestrationResult()).getUsage();
+  }
+
+  /**
+   * Get all messages.
+   *
+   * @return A list of all messages.
+   */
+  @Nullable
+  public List<ChatMessage> getAllMessages() {
```

Comment:
```suggestion
  public List<ChatMessage> getMessageHistory() {
```
debatable @MatKuhr @newtork

Reply:
Kept "getAllMessages".